src/core/basic_star_file_reader.cpp

```diff
@@ -395,6 +401,7 @@ bool BasicStarFileReader::ReadFile(wxString wanted_filename, wxString *error_str
 		if (current_line.StartsWith("_rlnMicrographName") == true) micrograph_name_column = current_column;
 		else
 		if (current_line.StartsWith("_rlnRandomSubset") == true) random_subset_column = current_column;
+		else if (current_line.StartsWith("_rlnImageName") == true) image_name_column = current_column;
```

Comment:
The whole point of the star file format is that it can contain anything.  If we were to do that we would have to explicilty list all the Relion headers, even the ones we don't use.  Then when they added a new one our code would start throwing warnings - so I think the way it is currently setup is the wanted behaviour - we grab the stuff we want, and ignore everything else.

Reply:
That makes sense